File: src/main/java/com/illuzionzstudios/mist/model/Pair.java
package com.illuzionzstudios.mist.model;

import lombok.Getter;

/**
 * Simple class that contains two values at once
 */
public class Pair<K, V> {

    /**
     * First value
     */
    @Getter
    private final K key;

    /**
     * Second value
     */
    @Getter
    private final V value;

    /**
     * Construct pair from 2 values
     */
    public Pair(K key, V value) {
        this.key = key;
        this.value = value;
    }

}

File: src/main/kotlin/com/illuzionzstudios/mist/model/Pair.kt
package com.illuzionzstudios.mist.model

import com.illuzionzstudios.mist.compatibility.ServerVersion.V

/**
 * Simple class that contains two values at once
 */
class Pair<K, V>
/**
 * Construct pair from 2 values
 */(
    /**
     * First value
     */
    val key: K,
    /**
     * Second value
     */
    val value: V
)